cmd/introspect/main.go

```diff
@@ -23,5 +26,25 @@ func main() {
 	if err != nil {
 		panic(err)
 	}
+
+	var schemaResp introspection.Response
+	if err := json.Unmarshal([]byte(res), &schemaResp); err != nil {
+		panic(fmt.Errorf("failed to unmarshal introspection JSON: %w", err))
+	}
+
+	schemaTypes := introspection.Types{}
+	for _, schemaType := range schemaResp.Schema.Types {
+		if strings.HasPrefix(schemaType.Name, "_") {
+			continue
+		}
```

Comment:
Note to the reviewer: this change is required because previously we were relying on graphql's typescript library to filter out `__` prefixes when creating a schema from an introspection result.

But, with the change to use `_` prefixes for "dagger internal" in some places, we need an extra processing step here that we didn't need before.